flaggr/src/main/java/com/comuto/flag/Flaggr.java

```diff
@@ -29,7 +30,7 @@
 
     private static final String TAG = "Flaggr";
 
-    protected static final String FLAGGR_SHARED_PREFERENCES_PREFIX = "FLAGGR_ID_";
+    protected static final String FLAGGR_SHARED_PREFERENCES_PREFIX = "FLAGGR_PREF_ID_";
```

Comment:
this change of preference key is necessary ? if not better to avoid it to not erase previous values (which maybe fine but just better to avoid the risk)